### ISSUE TITLE
alertText() message

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#380]
After the recent updates by @robnagler I get the following message template:
![alerttext](https://cloud.githubusercontent.com/assets/13209176/18638693/1bff7180-7e5f-11e6-856c-9497ec399cb3.png)

**Comment by mrakitin** [comment]
There is a bunch of such messages in the server log:
```js
10.0.2.2 - - [19/Sep/2016 15:51:53] "POST /error-logging HTTP/1.1" 200 -
../sirepo/server.py:192:app_error_logging 10.0.2.2: javascript error: {
    "cause": "",
    "message": "<no message>",
    "stackTrace": [
        "self.alertText@http://localhost:8000/static/js/sirepo.js?20160919.155152:140:9",
        ".controller/$scope.alertText@http://localhost:8000/static/js/sirepo-components.js?20160919.155152:123:24",
        "anonymous/fn@http://localhost:8000/static/js/ext/angular-1.4.2.min.js?20160919.155152 line 212 > Function:2:215",
        "f/<@http://localhost:8000/static/js/ext/angular-1.4.2.min.js?20160919.155152:116:1",
        "jf/this.$get</n.prototype.$digest@http://localhost:8000/static/js/ext/angular-1.4.2.min.js?20160919.155152:132:122",
        "jf/this.$get</n.prototype.$evalAsync/<@http://localhost:8000/static/js/ext/angular-1.4.2.min.js?20160919.155152:135:76",
        "e@http://localhost:8000/static/js/ext/angular-1.4.2.min.js?20160919.155152:43:179",
        "Kf/l.defer/c<@http://localhost:8000/static/js/ext/angular-1.4.2.min.js?20160919.155152:46:68"
    ],
    "type": "exception",
    "url": "http://localhost:8000/srw#/simulations"
}
```

**Comment by robnagler** [comment]
Strange error. I can't reproduce. Did you refresh your client? @moellep are
you seeing this?

**Comment by moellep** [comment]
Yes, I saw them. Removing the srdbg() calls from sirepo.js fixed it for me,
I think.

**Comment by robnagler** [comment]
Fix 467655ba9d9280eddf213b2ed8a735998580ccc4
